crates/biome_module_graph/src/js_module_info/ad_hoc_scope_resolver.rs

```diff
@@ -118,6 +122,54 @@ impl TypeResolver for AdHocScopeResolver {
         GLOBAL_RESOLVER.resolve_type_of(identifier)
     }
 
+    fn resolve_import(&mut self, qualifier: &TypeImportQualifier) -> Option<Type> {
```

Comment:
This implementation probably needs reverting. I think I was going into a dead end here.